sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -1,15 +1,30 @@
 # Release History
 
-## 11.6.0-beta.2 (Unreleased)
+## 11.6.0-beta.2 (2022-10-12)
```

Comment:
nit: since this is being released a day later than the PR submitted this will need to be

```suggestion
## 11.6.0-beta.2 (2022-10-13)
```